### PR TITLE
[WIP] Fix broken build on deprecated methods

### LINKED ISSUE #61: PApplet.selectFolder not found
Travis noticed that we broke the build on `PreferencesFrame`: https://travis-ci.org/sampottinger/processing4/builds/639584837?utm_medium=notification&utm_source=github_status.

### LINKED ISSUE #62: PGraphicsPDF build broken
Looks like PGraphicsPDF is failing:

```
PGraphicsPDF.java:431: error: cannot find symbol
    [javac]       g2.drawImage(image.getImage(), 0, 0, null);
```

### LINKED ISSUE #66: size call with PDF fails
Whoops! Looks like there was a regression on:

```
size(400, 400, PDF, "filename.pdf");
```

The transfer over of the 4th parameter was lost.

### PR DESCRIPTION
**Pull request by sampottinger**
It looks like some of the deprecations broken the build as caught by Travis (I can confirm on my local). This is resolving those issues as well as a regression noticed in preproc for pdf mode.

Resolves #61

Resolves #62

Resolves #66

### CODE REVIEW

**Comment by sampottinger** [comment]
Resolves https://github.com/processing/processing4/issues/61.

**Comment by sampottinger** [comment]
Resolves #66

File: app/src/processing/app/Language.java [hunk 1]
```diff
@@ -170,6 +170,18 @@ static public Language init() {
   }
 
 
+  static public void addModeStrings(Mode mode) {
```
Comment (benfry) [inline app/src/processing/app/Language.java:173]: Looks like some other code snuck in here? I haven't had time to verify the language support for Modes yet.

**Comment by benfry** [comment]
Turns out I was building some other stuff from the 3.x version of processing-core, so it caused all these. Sorting through 'em now…

**Comment by sampottinger** [comment]
Looks like you fixed most of these @benfry! I'll close this out.

**Pull request closed without merging**